src/components/CardDeck/styles.module.scss

```
@@ -41,6 +41,10 @@
     // siblings share the row width and crush each other on narrow screens.
     flex-direction: column;
     min-width: min(var(--card-deck-item-min-width, 320px), var(--card-deck-item-max-width, calc(100vw - 176px)));
+    // #568: without a max-width a growable item contributes its content
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)